test/units/modules/system/test_iptables.py

```diff
@@ -578,6 +578,65 @@ def test_insert_jump_reject_with_reject(self):
             'tcp-reset',
         ])
 
+    def test_jump_tee_gateway_negative(self):
+        """ Missing gateway when JUMP is set to TEE """
+        set_module_args({
+            'table': 'mangle',
+            'chain': 'PREROUTING',
+            'in_interface': 'eth0',
+            'protocol': 'udp',
+            'match': 'state',
+            'jump': 'TEE',
+            'ctstate': ['NEW'],
+            'destination_port': '9521',
+            'destination': '127.0.0.1'
+        })
+
+        try:
+            iptables.main()
```

Comment:
If the exception is not raised, the two asserts won't be evaluated and the test won't raise any error. You can instead use assertRaises() like in the other test:
(untested)
```python
with self.assertRaises(AnsibleFailJson) as e:
    iptables.main()
self.assertTrue(e.exception.args[0]['failed'])
self.assertEqual(e.exception.args[0]['msg'], 'jump is TEE but all of the following are missing: gateway')
```